src/wp-includes/block-supports/duotone.php

```diff
@@ -456,17 +453,52 @@ function wp_render_duotone_filter_preset( $preset ) {
 		$svg = trim( $svg );
 	}
 
-	add_action(
-		// Safari doesn't render SVG filters defined in data URIs,
-		// and SVG filters won't render in the head of a document,
-		// so the next best place to put the SVG is in the footer.
-		is_admin() ? 'admin_footer' : 'wp_footer',
-		function () use ( $svg ) {
-			echo $svg;
+	return $svg;
+}
+
+
+/**
+ * Registers the style and colors block attributes for block types that support it.
+ *
+ * @since 5.8.0
```

Comment:
```suggestion
 * @since 5.9.1
```

Reply:
This isn't a new function but might need to do a sanity check if it was introduced in 5.8.0

Reply:
This function was introduced in 5.8 - https://core.trac.wordpress.org/ticket/52991 & https://core.trac.wordpress.org/changeset/50929.